trieste/models/gpflow/sampler.py

```diff
@@ -91,11 +113,23 @@ def sample(self, at: TensorType, *, jitter: float = DEFAULTS.JITTER) -> TensorTy
         mean, var = self._model.predict(at[..., None, :, :])  # [..., 1, 1, L], [..., 1, 1, L]
         var = var + jitter
 
-        if not self._initialized:
-            self._eps.assign(
-                tf.random.normal([self._sample_size, tf.shape(mean)[-1]], dtype=tf.float64)
-            )  # [S, L]
+        def sample_eps() -> tf.Tensor:
             self._initialized.assign(True)
+            skip = IndependentReparametrizationSampler.skip
+            IndependentReparametrizationSampler.skip.assign(skip + self._sample_size)
+            normal_samples = idealised_normal_samples(
```

Comment:
Should we have the possibility to revert to regular sampling rather than QMC? as in keep it as default but if asked explicitely revert to basic MC.

Might be OK it ignore if it's complicated.